src/cmds/qstat.c

```diff
@@ -2889,26 +2915,28 @@ qstat -B [-f] [-F format] [-D delim] [ server_name... ]\n";
 				pbs_statfree(p_server);
 				p_server = NULL;
 				if (E_opt == 1)
-					connect = cnt2server(prev_server);
+					conn = cnt2server(prev_server);
 				else
-					connect = cnt2server(server_out);
+					conn = cnt2server(server_out);
 
-				if (connect <= 0) {
-					fprintf(stderr, "qstat: cannot connect to server %s (errno=%d)\n",
+				if (conn <= 0) {
+					fprintf(stderr, "qstat: cannot conn to server %s (errno=%d)\n",
```

Comment:
looks like the word connect got inadvertently changed to conn due to search and replace